leaf-ui/scripts/main.js

```diff
@@ -47,6 +49,14 @@ mode = "Modelling";		// 'Analysis' or 'Modelling'
 linkMode = "Relationships";	// 'Relationships' or 'Constraints'
 
 graph = new joint.dia.Graph();
```

Comment:
Why was their a graphObject and a graph before but now we only need a graph?
Does this any anything to do with the constraint view?